addon/components/bourbon-text-field.js

```diff
@@ -22,21 +22,21 @@ export default Component.extend({
   isFocused: false,
   noLabel: false,
 
-  fieldType: computed('type', function() {
+  fieldType: computed('type', function () {
```

Comment:
Not sure why spaces are being added here, seems inconsistent with the rest of the codebase.